package.json

```diff
@@ -6,8 +6,8 @@
     "email": "oss@atomist.com"
   },
   "keywords": [
-    "MONITOR",
-    "NOTIFICATIONS"
+    "DEVOPS",
+    "DEPLOY"
```

Comment:
So MONITOR maps to DEVOPS and NOTIFICATIONS to DEPLOY?

Reply:
Not really mapping, but changing categories. This is part of the [category redo](https://www.notion.so/atomist/Category-revamp-5217a24fb8744995a92fdea44bd93877). Optimizing for factors like groupings of > 1 skill.